applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bt_rend.h"
+
+#include <zephyr/zbus/zbus.h>
+
+#include "bt_volume_control.h"
+#include "nrf5340_audio_common.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_renderer, CONFIG_BT_RENDERER_LOG_LEVEL);
+
+ZBUS_CHAN_DEFINE(volume_chan, struct volume_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
```

Comment:
Could it be beneficial to have something with zbus in the channel name?
```suggestion
ZBUS_CHAN_DEFINE(zbus_vol_chan, struct volume_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
```

Reply:
I think that is, so called, butter on lard. This is a zbus chan define, and the name contains chan to indicate that it is a channel, and will only be used in zbus related function calls (i.e. zbus_chan_pub)